pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -629,13 +673,80 @@ func (b *OFBridge) RetryInterval() time.Duration {
 	return b.retryInterval
 }
 
+func (b *OFBridge) queryTableFeatures() {
```

Comment:
I think the code is a bit unintuitive right now, because `queryTableFeatures` also starts a goroutine which will process the response and in turn handle setting the table name

could I suggest the following:
```go
// queryTableFeatures queries table features from the switch using a multipart request message.
// Replies from the switch will written to a channel, to be handled by the provided replyHandler function in a separate goroutine.
func (b *OFBridge) queryTableFeatures(replyHandler func(ch <-chan *openflow13.MultipartReply)) {
    // ...
    go func() {
        defer delete(b.mpartRplChs, xid)
        replyHandler(ch)
    }
    // ...
}
```

would something like this work?